app/code/Magento/Catalog/Block/Adminhtml/Product/Edit/Tab/Options/Option.php

```diff
@@ -390,6 +371,63 @@ public function getOptionValues()
         return $this->_values;
     }
 
+    /**
+     * Get Option Value Of Group Select
+     *
+     * @param array $value
+     * @param \Magento\Catalog\Model\Product\Option $option
+     * @param boolean $showPrice
+     * @param int $scope
+     * @return array
+     */
+    public function getOptionValueOfGroupSelect($value, $option, $showPrice, $scope)
```

Comment:
This method should be `private` instead of `public` I think?

Reply:
yes. I have just changed it. Thanks.